OasisDEX/oasis-borrow
# Translation parameter fix

# [Translation parameter fix](https://www.notion.so/oazo/Bugs-and-Suggestions-3fa8565c715f4c31bcdeb07cc8e9e76b)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- translation parameter fix
  
## How to test 🧪
  <Please explain how to test your changes>
- using goerli and Automation feature flag, go to protection tab and click on Est. DAI / COLL on stop loss trigger detailed card. When switching between Close to Eth / Close to Dai text should be updated properly

<img width="756" alt="image" src="https://user-images.githubusercontent.com/39706811/164401067-1676c106-5263-4d4f-ac50-e31e77f4b467.png">
